CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Reflection.Metadata.Ecma335;
+
+namespace CanHazFunny;
+
+public class Jester
+{
+    public IJokeService JokeService { get; }
+    public IJokeOutput JokeOutput { get; }
+    public Jester(IJokeService? _jokeService,IJokeOutput? _jokeOutput)
```

Comment:
```suggestion
    public Jester(IJokeService jokeService, IJokeOutput jokeOutput)
```
If you are going to throw a null exception, don't mark the parameters as nullable. Marking them as nullable here says you think it's okay to pass null into your constructor.